src/state/megaStore.tsx

```diff
@@ -97,7 +97,7 @@ export const Provider: ParentComponent = (props) => {
         price: 0,
         fiat: localStorage.getItem("fiat_currency")
             ? (JSON.parse(localStorage.getItem("fiat_currency")!) as Currency)
-            : FIAT_OPTIONS[USD_INDEX],
+            : BTC_USD_OPTIONS[0],
```

Comment:
I really don't like this hardcoded indexing into an array it's just hard to read at the callsite and lame to maintain.

how I interpreted @benthecarman's suggestion:

in currencies.ts make a `export const USD_CURRENCY` item, `export const BTC_CURRENCY` item, then the currencies array is `[BTC_CURRENCY, USD_CURRENCY, ...rest]`. now you can import BTC or USD items as values instead of indexing them. and there's no need to concat or sort the array